src/views/AgmLocationChg.vue

```diff
@@ -454,6 +455,12 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
     // see https://v3.vuejs.org/api/options-lifecycle-hooks.html#mounted
     await this.$nextTick()
 
+    // Pre-populate the certified block with the logged in user's name (if not staff)
+    if (!this.isRoleStaff) {
+      const userInfo = await AuthServices.fetchUserInfo(this.getAuthApiUrl).then(response => response?.data)
+      this.certifiedBy = userInfo?.firstname + ' ' + userInfo?.lastname
```

Comment:
Are you sure we didn't already load this data in App.vue?

Reply:
We use this same function in App.vue but nothing is being set in the store. I had two choices, either set the logged in user's name in the store or add this here. I went with this route. Let me know if I should go with the first one Sev (or if you think something else is better).

Reply:
It's better for an API call to be made as few times as possible.

Also, keeping it here could slow down the page's loading by up to 30 seconds (or whatever the network timeout is), and I was hoping we could get rid of the loading spinner and associated plumbing.